src/galileo/job_progress.py

```diff
@@ -0,0 +1,115 @@
+import logging
+import random
+from time import sleep
+from typing import Optional
+
+from pydantic import UUID4
+from tqdm.auto import tqdm
+
+from galileo.config import GalileoPythonConfig
+from galileo.resources.api.jobs import (
+    get_job_jobs_job_id_get,
+    get_jobs_for_project_run_projects_project_id_runs_run_id_jobs_get,
+)
+from galileo.resources.models import HTTPValidationError, JobDB
+from galileo_core.constants.job import JobStatus
+from galileo_core.constants.scorers import Scorers
+
+_logger = logging.getLogger(__name__)
+
+
+def get_job(job_id: UUID4) -> JobDB:
+    config = GalileoPythonConfig.get()
+
+    response = get_job_jobs_job_id_get.sync(client=config.api_client, job_id=str(job_id))
+
+    if isinstance(response, HTTPValidationError):
+        raise ValueError(response.detail)
+    if not response:
+        raise ValueError(f"Failed to get job status for job {job_id}")
+    return response
+
+
+def get_run_scorer_jobs(project_id: UUID4, run_id: UUID4) -> list[JobDB]:
+    config = GalileoPythonConfig.get()
+
+    response = get_jobs_for_project_run_projects_project_id_runs_run_id_jobs_get.sync(
+        client=config.api_client, project_id=str(project_id), run_id=str(run_id)
+    )
+    if isinstance(response, HTTPValidationError):
+        raise ValueError(response.detail)
+    if response is None:
+        raise ValueError(f"Failed to get scorer jobs for project {project_id}, run {run_id}")
+
+    _logger.info(f"Scorer jobs: {response}")
+
+    # TODO Source (rungalileo-dev): Authorization error accessing https://us-python.pkg.dev/rungalileo-dev/rungalileo/simple/rungalileo/
```

Comment:
How to install rungalileo? Or should I move `JobName` to Core?

Reply:
When i try to pip install `rungalileo` locally.. but looks like I might actually move that enum from `rungalileo` to `core` so might be a moot point